packages/phone-number-privacy/signer/src/database/models/domain.ts

```diff
@@ -0,0 +1,20 @@
+export const DOMAINS_TABLE = 'domains'
+export enum DOMAINS_COLUMNS {
+  domain = 'domain',
+  counter = 'counter',
+  timer = 'timer',
+  disabled = 'disabled',
+}
+export class Domain {
+  [DOMAINS_COLUMNS.domain]: string;
+  [DOMAINS_COLUMNS.counter]: number;
+  [DOMAINS_COLUMNS.timer]: number;
+  [DOMAINS_COLUMNS.disabled]: boolean
+
+  constructor(domain: string, counter: number, timer: number, disabled: boolean) {
```

Comment:
We should probably hardcode disabled to false here, since we should never be constructing a disabled Domain